django/contrib/admin/tests.py

```diff
@@ -14,11 +15,41 @@ def process_response(self, request, response):
         return response
 
 
+class AdminSeleniumMetaClass(type):
+    def __new__(cls, name, bases, attrs):
+        """
+        Dynamically inject new methods for running tests in different browsers
+        when multiple browser specs are provided (e.g. --selenium=firefox,chrome).
+        """
+        all_specs = os.environ.get('DJANGO_SELENIUM_SPECS', '').split(',')
+        if all_specs:
+            for key, value in attrs.items():
+                if isinstance(value, types.FunctionType) and key.startswith('test'):
+                    func = value
+                    for index, spec in enumerate(all_specs):
+                        new_func_name = '%s__%s' % (key, spec.replace('/', '_'))
```

Comment:
Oh I see now, this is why `setUpClass` was moved to `setUp` in the original patch. Anyway, I think simpler is better for now so let's remove the metaclass. Maybe you can keep this around in a branch somewhere so we can get it back if we decide to at some point.